app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
+    def __init__(self, nickname: str, favourite_dish: str) -> None:
```

Comment:
Place all arguments in the `__init__` on a new line to preserve the overall project code style.